pandas/tests/frame/test_operators.py

```diff
@@ -237,7 +237,7 @@ def test_modulo(self):
         s = p[0]
         res = s % p
         res2 = p % s
-        assert not np.array_equal(res.fillna(0), res2.fillna(0))
```

Comment:
so another way to do this and maybe cleaner is to have a parameter ``compare='equal|'not_equal'``
The main issue I have with this method is the error messages are not as good (though maybe it doesn't matter)

Reply:
IIUC, wouldn't I have to propagate that parameter across every single `assert_*` function?  Also, I'm not sure I follow what error message you would expect.  If the `assert_*` function passes when it isn't supposed to, you can't specify what was "different" in this case.

Reply:
@jreback : If you have any other thoughts on this point, let me know.  Otherwise, I think this should be good to go.

Reply:
@jreback : Any thoughts?